tests/ManageCourses.Tests/DbIntegration/DataServiceExportTests.cs

```diff
@@ -1,47 +1,46 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using FluentAssertions;
 using GovUk.Education.ManageCourses.Api.Model;
 using GovUk.Education.ManageCourses.Api.Services;
 using GovUk.Education.ManageCourses.Api.Services.Data;
-using GovUk.Education.ManageCourses.Domain.DatabaseAccess;
 using GovUk.Education.ManageCourses.Tests.UnitTesting.Helpers;
 using Microsoft.Extensions.Logging;
 using Moq;
 using NUnit.Framework;
 
-namespace GovUk.Education.ManageCourses.Tests.UnitTesting
+namespace GovUk.Education.ManageCourses.Tests.DbIntegration
 {
     [TestFixture]
-    class DataServiceExportTests
+    class DataServiceExportTests : DbIntegrationTestBase
     {
-        private ManageCoursesDbContext _dbContext;
         private DataService _dataService;
 
-        [OneTimeSetUp]
-        public void Setup()
+        protected override void Setup()
         {
-            _dbContext = TestHelper.GetFakeData(EnumTestType.DataService);            
+            new TestHelper(Context).BuildFakeDataForService();
```

Comment:
new it up and don't use it?

Reply:
It's used immediately by calling BuildFakeDataForService